integrations/hello-world-ng7/src/app/app.component.spec.ts

```diff
@@ -12,9 +13,10 @@ describe('NGXS', () => {
     class CountriesState {}
 
     // Act
-    TestBed.configureTestingModule({
-      imports: [NgxsModule.forRoot([CountriesState])]
-    });
+    await TestBed.configureTestingModule({
+      imports: [NgxsModule.forRoot([CountriesState])],
+      providers: [Injector]
```

Comment:
Why the `Injector` is provided here?

Reply:
Good question. This spec was failing without this provider back then when I was refactoring. Idk why it was and way it is not now. Maybe I missed something. I'll remove it in a moment.